docs/jwst/undersampling_correction/description.rst

```diff
@@ -29,23 +30,22 @@ form of a `~jwst.datamodels.RampModel`.
 
 
 Algorithm
----------
-The algorithm for this step is to flag as UNDERSAMP the first group exceeding (and all
-subsequent groups) the value of the ``signal_threshold`` parameter, which has units of
-ADU. Those groups will also be flagged as DO_NOT_USE, so that they will not be included
-in the slope calculation in the ``ramp_fitting`` step. The motivation for this step is
-to have the readnoise variance (calculated in ``ramp_fitting``) be similar to pixels
-unaffected by charge migration. Despite being also flagged as DO_NOT_USE, those
-UNDERSAMP groups will be included in the calculation of the readnoise variance by ignoring the
-DO_NOT_USE flag. For the Poisson noise variance calculation in ``ramp_fitting``, the
-UNDERSAMP/DO_NOT_USE groups will not be included. As detailed in the ``ramp_fitting``
-documentation, a modified version of the readnoise variance is used for calculations requiring
-combining variances,
+--------- 
+The first group, and all subsequent groups, exceeding the value of the 
+``signal_threshold`` parameter is flagged as UNDERSAMP. ``signal_threshold`` is in units 
+of ADUs. These groups will also be flagged as DO_NOT_USE, and will not 
+be included in the slope calculation during the ``ramp_fitting`` step. Despite being flagged 
+as DO_NOT_USE, these UNDERSAMP groups are still included in the calculation of the
+variance due to readnoise. 
+This results in a readnoise variance for undersampled pixels that is similar to that of 
+pixels unaffected by charge migration. For the Poisson noise variance calculation in 
+:ref:`ramp_fitting <ramp_fit>`, the UNDERSAMP/DO_NOT_USE groups are not included.
```

Comment:
same issue as above, where the second field (in angle brackets) needs to be "ramp_fitting_step".